osu.Game.Rulesets.Taiko/Judgements/TaikoDrumRollTickJudgement.cs

```diff
@@ -9,7 +9,9 @@ namespace osu.Game.Rulesets.Taiko.Judgements
 {
     public class TaikoDrumRollTickJudgement : TaikoJudgement
     {
-        public override HitResult MaxResult => HitResult.SmallTickHit;
+        public bool IsBonus = false;
+
+        public override HitResult MaxResult => IsBonus ? HitResult.LargeBonus : HitResult.SmallTickHit;
```

Comment:
```suggestion
        public override HitResult MaxResult => IsBonus ? HitResult.LargeBonus : base.MaxResult;
```

Reply:
Does this work? Isn't `base.MaxResult = HitResult.Great`? (https://github.com/sw1tchbl4d3r/osu/blob/a5bf16e87345d488ff5f95578002a9be29276f47/osu.Game.Rulesets.Taiko/Judgements/TaikoJudgement.cs#L13)

Reply:
> Does this work? Isn't `base.MaxResult = HitResult.Great`?
> 
> https://github.com/ppy/osu/blob/a5bf16e87345d488ff5f95578002a9be29276f47/osu.Game.Rulesets.Taiko/Judgements/TaikoJudgement.cs#L13

I think move it to `TaikoHitObject` makes sense

Reply:
I'm resolving this review conversation because the original suggestion is both overly nitpicky, and - as noted by @sw1tchbl4d3r - just plain wrong.